docs/06-tutorials/02-todolist.md

```diff
@@ -0,0 +1,475 @@
+---
+title: Deploying a dApp using the Remix IDE
+---
+ 
+This tutorial shows how to create a smart contract and a dApp using ReactJS, where the contract and the dApp will interact with each other. This two-part series discusses how to deploy a smart contract using the Remix IDE then develop the To Do List app using ReactJS.
+
+#### What is a Smart Contract?
+
+A smart contract is a program that runs on the blockchain. It is a digital contract and agreement that requires no human interaction nor trust from a third party. The smart contract in this exercise is written in Solidity. Smart contracts are executed on an EVM (a virtual machine).
+
+#### What is a dApp?
+
+A dApp (decentralized application) is an application that runs on many servers across the blockchain. dApps are mostly used in the crypto-marketplace, where they require security and transparency.
+
+## Before You Begin
+
+To start this exercise you need to have the following apps/tools installed or available in your localhost:
+
+* **Remix IDE** - development tool to compile and deploy the smart contract. You can use the online version of the Remix IDE.
+
+   **Note:** Read the Remix documentation for usage.
+
+* **Node.js** - a programming environment needed to create the dApp (installation required).
+
+* **Gobi Testnet (current EON network)** - the EVM sidechain where the smart contract and dApp are deployed to.
+
+### Gobi Testnet Configuration
+
+**Network Name:** Gobi Testnet
+**RPC URL:** https://gobi-testnet.horizenlabs.io/ethv1
+**Chain ID:** 1663 

```

Comment:
Make sure these each appear on their own line

Reply:
added spaces between lines

Reply:
Would it make more sense if we turn this block into `code` @Ramone-HL like so? 

<img width="577" alt="Screenshot 2023-05-09 at 10 05 49 PM" src="https://github.com/HorizenOfficial/evm-documentation/assets/5024180/adc60a84-1c76-4b65-a0ce-e5707009fb84">